content/influxdb/v2.0/write-data/troubleshoot.md

```diff
@@ -16,44 +16,49 @@ related:
 ---
 Learn how to handle and recover from errors when writing to InfluxDB.
 
-- [Discover common failure scenarios](#common-failure-scenarios)
-- [HTTP status codes](#http-status-codes)
+- [Discover common failure scenarios](#discover-common-failure-scenarios)
+- [Review HTTP status codes](#review-http-status-codes)
 - [Troubleshoot failures](#troubleshoot-failures)
 
-## Common failure scenarios
+## Discover common failure scenarios
```

Comment:
I guess this doc should be updated too, as it looks mostly like a dupe of the other `troubleshoot.md` doc.

Reply:
> I guess this doc should be updated too, as it looks mostly like a dupe of the other `troubleshoot.md` doc.

@rogpeppe The feature is only in Cloud still, right?

Reply:
I'm not precisely sure about that, but if it's the case, you'll perhaps want to remove the rejected points changes that you've added in this PR?